### PR TITLE
DOCS-1936: Review navigate with rover base tutorial R2D2

### PR DESCRIPTION
**Pull request by skyleilani**

### CODE REVIEW

File: docs/tutorials/services/navigate-with-rover-base.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ We used a [`jetson` board](/components/board/jetson/), but you can use any model
 ![Configuration of a jetson board with digital interrupts in the Viam app config builder.](/tutorials/navigate-with-rover-base/board-config-builder.png)
 
 2. Configure [digital interrupts](/components/board/#digital_interrupts) on your board to signal precise GPIO state changes to the [encoders](/components/encoder/) on your rover base.
-   Copy and paste the following into your board's **Attributes** to add [digital interrupts](/components/board/#digital_interrupts) on pins `31`, `29`, `23`, and `21`:
+   Copy and paste the following into your board's attributes box to add [digital interrupts](/components/board/#digital_interrupts) on pins `31`, `29`, `23`, and `21`:
```
Comment (npentrel) [inline docs/tutorials/services/navigate-with-rover-base.md:101]: Box isn't the correct term. It's a textarea I think but no one says that, field works I think?
```suggestion
   Copy and paste the following into your board's **Attributes** field to add [digital interrupts](/components/board/#digital_interrupts) on pins `31`, `29`, `23`, and `21`:
```

File: docs/tutorials/services/navigate-with-rover-base.md [hunk 2]
```diff
@@ -421,12 +430,13 @@ In the **Raw JSON** mode in your machine's **Config** tab, add the following JSO
...
-3. Then click **Create Service**.
-4. Select **Raw JSON** mode.
-   Copy and paste the following into your new service's `"attributes"`:
+1. Navigate to the **CONFIGURE** tab of your machine's page in the [Viam app](https://app.viam.com).
+1. Click the **+** icon next to your machine part in the left-hand menu and select **Service**.
+1. Select the `Navigation` type.
+1. Enter a name or use the suggested name for your service and click **Create**.
+
+1. Select **JSON** mode.
+   Copy and paste the following into your new service's attributes box:
```
Comment (npentrel) [inline docs/tutorials/services/navigate-with-rover-base.md:439]:
```suggestion
   Copy and paste the following into your new service's **Attributes** field:
```

**Review by npentrel — COMMENTED** [review]
What state is this in? It looks like it might be ready @skyleilani ?

File: docs/tutorials/services/navigate-with-rover-base.md [hunk 3]
```diff
@@ -421,12 +430,13 @@ In the **Raw JSON** mode in your machine's **Config** tab, add the following JSO
...
 
-1. On your machine's **Config** page, navigate to the **Services** tab.
-2. At the bottom of the page, create a service.
-   Choose `Navigation` as the type.
-3. Then click **Create Service**.
-4. Select **Raw JSON** mode.
-   Copy and paste the following into your new service's `"attributes"`:
+1. Navigate to the **CONFIGURE** tab of your machine's page in the [Viam app](https://app.viam.com).
+1. Click the **+** icon next to your machine part in the left-hand menu and select **Service**.
+1. Select the `Navigation` type.
```
Comment (sguequierre) [inline docs/tutorials/services/navigate-with-rover-base.md:435]:
```suggestion
1. Select the `navigation` type.
```
All services are lowercase now

File: docs/tutorials/services/navigate-with-rover-base.md [hunk 4]
```diff
@@ -480,20 +490,22 @@ In the **Raw JSON** mode in your machine's **Config** tab, add the following JSO
...
-You can add waypoints from the [Control tab](#control-tab-method) or [programmatically](#programmatic-method).
+You can add waypoints from the [**CONTROL** tab](#control-tab-method) or [programmatically](#programmatic-method).
 
 ### Control tab method
 
-Go to the **Control** tab of your robot in the [Viam app](https://app.viam.com), and open the **navigation** card.
-From there, ensure that **Navigation mode** is selected as **Manual**, so your robot will not begin navigation while you add waypoints.
+Go to the **CONTROL** tab of your robot in the [Viam app](https://app.viam.com), and open the navigation service card.
+
+- Is there anything in place of this they will need to do in order to ensure the robot will nto begin navigation while they add waypoints? Since there is no longer navigation mode?
```
Comment (sguequierre) [inline docs/tutorials/services/navigate-with-rover-base.md:507]:
```suggestion
```

File: docs/tutorials/services/navigate-with-rover-base.md [hunk 5]
```diff
@@ -510,8 +522,8 @@ Hover over a waypoint in the left-hand menu and click the trash icon to delete a
 #### (Optional) Add obstacles
 
 If you want your robot to avoid certain obstacles in its path while navigating, you can also add obstacles.
-Select **Obstacles** on the upper-left corner menu of the navigation card.
-Zoom in on your current location and click on the map to add an obstacle.
+In the **CONFIGURE** tab, select the **Obstacles** subtab on the navigation card.
+Zoom in on your current location, then hold shift and drag on the map to draw an obstacle.
```
Comment (sguequierre) [inline docs/tutorials/services/navigate-with-rover-base.md:526]:
```suggestion
Zoom in on your current location, then hold **shift** and drag on the map to draw an obstacle.
```

File: docs/tutorials/services/navigate-with-rover-base.md [hunk 6]
```diff
@@ -98,7 +98,7 @@ We used a [`jetson` board](/components/board/jetson/), but you can use any model
 ![Configuration of a jetson board with digital interrupts in the Viam app config builder.](/tutorials/navigate-with-rover-base/board-config-builder.png)
 
 2. Configure [digital interrupts](/components/board/#digital_interrupts) on your board to signal precise GPIO state changes to the [encoders](/components/encoder/) on your rover base.
-   Copy and paste the following into your board's **Attributes** to add [digital interrupts](/components/board/#digital_interrupts) on pins `31`, `29`, `23`, and `21`:
+   Copy and paste the following into your board's attributes field to add [digital interrupts](/components/board/#digital_interrupts) on pins `31`, `29`, `23`, and `21`:
```
Comment (sguequierre) [inline docs/tutorials/services/navigate-with-rover-base.md:101]:
```suggestion
   Find your board on the **CONFIGURE** tab in **Builder** mode. Click the **{}** (Switch to advanced) button on the right side of your board's card to switch to JSON attributes editing mode.
   Copy and paste the following JSON into your board's attributes field to add [digital interrupts](/components/board/#digital_interrupts) on pins `31`, `29`, `23`, and `21`:
```
This changed slightly with the new board cards and it's now harder to find where to do it

File: docs/tutorials/services/navigate-with-rover-base.md [hunk 7]
```diff
@@ -421,12 +432,13 @@ In the **Raw JSON** mode in your machine's **Config** tab, add the following JSO
...
-2. At the bottom of the page, create a service.
-   Choose `Navigation` as the type.
-3. Then click **Create Service**.
-4. Select **Raw JSON** mode.
-   Copy and paste the following into your new service's `"attributes"`:
+1. Navigate to the **CONFIGURE** tab of your machine's page in the [Viam app](https://app.viam.com).
+1. Click the **+** icon next to your machine part in the left-hand menu and select **Service**.
+1. Select the `navigation` type.
+1. Enter a name or use the suggested name for your service and click **Create**.
+
```
Comment (sguequierre) [inline docs/tutorials/services/navigate-with-rover-base.md:439]:
```suggestion
```

**Review by sguequierre — APPROVED** [review]
LGTM % 1 suggestions!

**Pull request merged**